packages/teams-js/src/public/dialog.ts

```diff
@@ -225,4 +228,43 @@ export namespace dialog {
       return runtime.supports.dialog ? (runtime.supports.dialog.bot ? true : false) : false;
     }
   }
+
+  /**
+   * @hidden
+   * Hide from docs
+   * --------
+   * Convert UrlDialogInfo to DilogInfo to send the information to host in {@linkcode open} API.
```

Comment:
"DialogInfo" instead of "DilogInfo"

Reply:
Thank you for catching it. I need to pay more attention to spellings and grammar 😣